When the websocket is broken gracefully reload the site.

johnrtipton opened the pull request:
When the websocket is broken, the site currently displays an alert dialog asking the user to reload the page.

This patch gracefully tries to reload the site, backing off to 1 minute per retry.

elimintz commented:
Thank you for this, looks good. Could you please explain why you also removed lines 76 to 78 in main.html?

knoxvilledatabase commented:
When I put lines 76 to 78 back, the code gets stuck in a loop.

johnrtipton commented:
> Thank you for this, looks good. Could you please explain why you also removed lines 76 to 78 in main.html?

Originally, I the way I approached it I needed `ws_url` so I rewrote that a bit.  Turns out I didn't need it, but I thought it i was a little more concise.

WolfgangFahl commented:
should this go into 0.0.3?
